pkg/plugin/windows/pktmon/types.go

```diff
@@ -0,0 +1,39 @@
+//go:build experimental	
+
+package pktmon
+
+import (
+	"net"
+
+	"github.com/cilium/cilium/api/v1/flow"
+)
+
+type PktMon interface {
+	Initialize() error
+	GetNextPacket() (*Packet, *Metadata, error)
+}
+
+// surely there is a better struct around the repo that is the same as this
+type Packet struct {
+	SourceIP   net.IP
+	DestIP     net.IP
+	SourcePort uint32
+	DestPort   uint32
+	Protocol   uint8
+	Direction  string // we currently don't set this, but it is available, check pktmon notes for Direction enum
+	syn        uint16
+	ack        uint16
+	fin        uint16
+	rst        uint16
+	psh        uint16
+	urg        uint16
```

Comment:
Should all of these bits be in `uint16`s or is it more hassle than it's worth to pack them into a byte?

Reply:
yeah we can optimize, [I really don't like this struct as a whole](https://github.com/microsoft/retina/pull/235/files/dd46db6d302b1148c5f06b9640d1e01542b26843#diff-ad926a15b40b4833e0be15cff18f8cb700152eca895eb838551b59b4f1eb6f24R16), and I would like to get rid of it in favor of something we might be using elsewhere, but keeping this struct scoped to the experiment for now


also you might notice that this pr [has a lot of C missing...](https://github.com/microsoft/retina/pull/235/files#diff-cfde01d9be83dc181dcc5d73b0d6607592f397ca2160f47c02e136b46659ba46R56-R57)

Reply:
Okay, that sounds fine for an experiment :)